### DISCUSSION TITLE
Maybe a extension that can do these things ?

### DISCUSSION THREAD

**Discussion by Mickmick21** [TurboWarp/extensions#228] [Ideas]
The things the extension could do is : delete sprites, add / remove variables, add / remove lists, remove sounds, and remove costumes. I have seen an extension to delete sprites, but it's very buggy and in a fork (not a URL)

**Comment by softedco** [comment]
> The things the extension could do is : delete sprites, add / remove variables, add / remove lists, remove sounds, and remove costumes. I have seen an extension to delete sprites, but it's very buggy and in a fork (not a URL) 

I get adding/removing variables/lists but why would you need the rest?

**Reply by Mickmick21** [reply]
delete sprites cuz ShovelUtilis can IMPORT them from a URL (or data uri), but what of getting rid of the imported sprite ?

**Reply by Mickmick21** [reply]
same for the others

**Comment by Flappy25** [comment]
> The things the extension could do is : delete sprites, add / remove variables, add / remove lists, remove sounds, and remove costumes. I have seen an extension to delete sprites, but it's very buggy and in a fork (not a URL) 

I don't think this would help much because you don't need to delete assets from the editor. Just hide them or make them stop working.
